HedgeModManager/Languages/pt-PT.xaml

```diff
@@ -70,17 +70,17 @@
     <system:String x:Key="MainUIInstallFormHeader"   >Instalar um mod através de...</system:String>
     <system:String x:Key="MainUIInstallFormOptionDir">Instalar de uma pasta</system:String>
     <system:String x:Key="MainUIInstallFormOptionArc">Instalar de um arquivo</system:String>
-    <system:String x:Key="MainUIInstallFormOptionNew">Criar um (para desenvolvedores!)</system:String>
```

Comment:
Keep "desenvolvedores", as mod makers are developers and not programmers

Reply:
You're right, not all mods were programmed and as such we can keep the original word